src/main.js

```diff
@@ -6,11 +6,31 @@ import vuetify from "./plugins/vuetify";
 import axios from "axios";
 import VueAxios from "vue-axios";
 
-Vue.use(VueAxios, axios)
+Vue.use(VueAxios, axios);
 
 axios.defaults.baseURL = "https://developers.zomato.com/api/v2.1";
 axios.defaults.headers.common["user-key"] = process.env.VUE_APP_API_KEY;
 
+//interceptors
+
+axios.interceptors.response.use(
+  response => {
+    if (response.status === 403) {
+      console.log(response);
+    }
+    return response;
+  },
+  error => {
+    if (error.response.status === 403) {
+      alert("please use your API key to run the app");
```

Comment:
What if I did and yet got 403? Tokens and keys can expire. Not to mention you're not being exactly nice to the user --- do you really expect they read the WHOLE readme before using your site? Well, now they need to!

Note that `alert` windows can be quite big and still remain readable... and you can copy-paste from them!